packages/ui/src/2_molecules/AmountInput/AmountInput.module.css

```
@@ -43,11 +43,19 @@
 }
 
 .invalid {
-  @apply outline outline-1 outline-error-light bg-gray-60;
+  @apply outline outline-1 outline-error-light bg-gray-70;
```

```suggestion
  @apply outline outline-1 outline-error-light bg-gray-60;
```
Figma shows color as #333845 for invalid state, but it's #2c303b in storybook